frontend/ui/network/manager.lua

```diff
@@ -5,6 +5,7 @@ local InfoMessage = require("ui/widget/infomessage")
 local LuaSettings = require("luasettings")
 local UIManager = require("ui/uimanager")
 local _ = require("gettext")
+local FFIUtil = require("ffi/util")
```

Comment:
Not quite. I gave the full correct order in my original comment https://github.com/koreader/koreader/pull/3482#discussion_r161472520.

```lua
local ConfirmBox = require("ui/widget/confirmbox")
local DataStorage = require("datastorage")
local Device = require("device")
local FFIUtil = require("ffi/util")
local InfoMessage = require("ui/widget/infomessage")
local LuaSettings = require("luasettings")
local UIManager = require("ui/uimanager")
local _ = require("gettext")
local T = FFIUtil.template
```

However, after @poire-z 's comment I've decided I'd prefer it to be changed like this:

```lua
local ConfirmBox = require("ui/widget/confirmbox")
local DataStorage = require("datastorage")
local Device = require("device")
local InfoMessage = require("ui/widget/infomessage")
local LuaSettings = require("luasettings")
local UIManager = require("ui/uimanager")
local ffiutil = require("ffi/util")
local _ = require("gettext")
local T = ffiutil.template
```